dll/win32/url/url_main.c

```diff
@@ -105,7 +105,9 @@ void WINAPI FileProtocolHandlerA(HWND hWnd, HINSTANCE hInst, LPCSTR pszUrl, int
  */
 void WINAPI OpenURLA(HWND hwnd, HINSTANCE inst, LPCSTR cmdline, INT show)
 {
-    FIXME("(%p, %p, %s, %d): stub!\n", hwnd, inst, debugstr_a(cmdline), show);
+    TRACE("(%p, %p, %s, %d): stub!\n", hwnd, inst, debugstr_a(cmdline), show);
+    if (PathIsURLA(cmdline))
+        ShellExecuteA(hwnd, NULL, cmdline, NULL, NULL, show);
```

Comment:
I would suggest you check what does Windows' url.dll does there, and whether it uses instead the
`void WINAPI OpenURL(HWND hWnd, HINSTANCE hInst, LPCSTR lpcstrUrl, int nShowCmd)` from ieframe.dll .... or whether it is the other way around.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/0f8c690c-7f15-4b53-9e9e-a4a72b4bbb7e)
Just calling `ShellExecuteA`. `OpenURL` is an alias of `OpenURLA`.